src/components/TransactionItem/index.tsx

```diff
@@ -54,6 +62,26 @@ const Time: React.FC<{ tx?: Transaction }> = ({ tx }) => {
   return null
 }
 
+const computeRGBPPCellAmount = (cells: Cell[]) => {
+  return cells.reduce((cur, cell) => {
+    try {
+      const script = addressToScript(cell.addressHash)
+      if (!script) {
```

Comment:
In what case will `script` be `falsy`

Reply:
I though that the `addressToScript` could return `false`. I just removed it